internal/pkg/pubsub/source_loader.go

```diff
@@ -125,73 +123,41 @@ func (s *SourceLoader) fetchProjectSources(ctx context.Context) error {
 	return nil
 }
 
-func (s *SourceLoader) handler(ctx context.Context, source *datastore.Source, msg string) error {
-	txn, innerCtx := apm.StartTransaction(ctx, fmt.Sprintf("%v handler", source.Name))
```

Comment:
@jirevwe Any reason why we removed this?

Reply:
That was a mistake, I'll add it back